### DISCUSSION TITLE
Sons Of The Forest

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#453] [Game request]
steam: https://store.steampowered.com/app/1326470/Sons_Of_The_Forest/

Dedicated Server Configuration Guide:
https://steamcommunity.com/sharedfiles/filedetails/?id=2992700419&snr=1_5_9_

**Comment by ad3d0** [comment]
Sons Of The Forest currently do not have dedicated server files released.

**Comment by ssiwm** [comment]
Soon™

**Comment by ssiwm** [comment]
updated with information about dedicated servers 

Dedicated Server Configuration Guide:
https://steamcommunity.com/sharedfiles/filedetails/?id=2992700419&snr=1_5_9_

**Comment by mvaessen** [comment]
Created development item #472
